app/src/main/res/values/colors.xml

```diff
@@ -132,4 +132,6 @@
     <color name="quick_action_pull_tab">#2915141A</color>
     <color name="quick_action_background_normal_theme">@color/toolbar_normal_theme</color>
     <color name="quick_action_background_private_theme">@color/toolbar_dark_mode</color>
+
+    <color name="fenix_snackbar_background">#232749</color>
```

Comment:
Do we want this to look the same in all themes?

Reply:
Good question. I'll ask @brampitoyo and follow up with a new PR :+1: